src/languageTag.ts

```diff
@@ -1,6 +1,17 @@
-export function normalizeLanguageTag(tagString: string) {
-  const match = /^([a-z]{2})(-[a-z]{2})?$/i.exec(tagString);
-  if (match === null) return null;
-  const [, language, region] = match;
-  return language.toLowerCase() + (region || '').toUpperCase();
+export const supportedTags = [
```

Comment:
should we have the supported tags correlated with what we have in companion-gettext? Or does it make sense to keep them decoupled?

Reply:
Ideally they'd be shared. We could export them from gettext and import here, but that doesn't seem ideal. I'd have liked to have made the FbOS 3 specific changes I made in gettext here, but it's tricky because all the code on this side is within a string.